### PR TITLE
Don't clone Cluster API

### PR DESCRIPTION
**Pull request by fmuyassarov**
Don't clone Cluster-api since we are not building clusterctl from CAPI code anymore.

### CODE REVIEW

**Review by namnx228 — COMMENTED** [review]
lgtm

**Comment by jan-est** [comment]
lgtm

File: lib/common.sh [hunk 1]
```diff
@@ -105,7 +104,6 @@ IPAM_BASE_URL="${IPAM_BASE_URL:-metal3-io/ip-address-manager}"
 IPAMREPO="${IPAMREPO:-https://github.com/${IPAM_BASE_URL}}"
 IPAMBRANCH="${IPAMBRANCH:-master}"
 
-CAPIPATH="${CAPIPATH:-${M3PATH}/cluster-api}"
 CAPI_BASE_URL="${CAPI_BASE_URL:-kubernetes-sigs/cluster-api}"
 CAPIREPO="${CAPIREPO:-https://github.com/${CAPI_BASE_URL}}"
```
Comment (furkatgofurov7) [inline lib/common.sh:108]: It seems to me, we can remove `CAPIREPO` too, since we are not using it anywhere, WDYT?

**Review by furkatgofurov7 — COMMENTED** [review]
Quesion inline

File: lib/common.sh (hunk 1 above)
Comment (fmuyassarov) [reply lib/common.sh]: Done.

**Review by maelk — APPROVED** [review]

**Review by furkatgofurov7 — COMMENTED** [review]
/lgtm

**Pull request merged**